contrib/examples/script/example.go

```diff
@@ -0,0 +1,99 @@
+package script
```

Comment:
```suggestion
// SPDX-License-Identifier: Apache-2.0                                                                                                                                                                                                                         
// Copyright Authors of Cilium                                                                                                                                                                                                                                 

package script
```

Reply:
I on purpose didn't add these as they're included directly into the docs. Must we have them?